apps/frontend/src/app/Components/Artifact/RVSlide.tsx

```diff
@@ -45,28 +58,58 @@ export default function RVSlide({
         overflow: 'visible',
       }}
     >
-      <CustomNumberInput
-        value={sliderLow}
-        onChange={(val) => setLow(clamp(val ?? 0, 0, levelHigh))}
-        sx={{ px: 1, pl: 2, width: 100, borderRadius: '4px 0 0 4px' }}
-        inputProps={{ sx: { textAlign: 'right' } }}
-        startAdornment={'RV: '}
-        disabled={disabled}
-      />
+      <Box
+        sx={{ width: 100, height: 30, display: 'flex', alignItems: 'center' }}
+      >
+        <Button
+          onClick={() =>
+            setuseMaxRV((useMaxRV) => {
+              reset()
+              return !useMaxRV
+            })
+          }
+          sx={{
+            p: 0,
+            minWidth: 0,
+            width: '50%',
+            height: '100%',
+            borderRadius: '4px 0 0 4px',
+            transition: 'transform 0.3s ease',
+            '&:hover': { transform: 'translate(-3px, -3px)' },
```

Comment:
I am not sure about the additional hover effect on this button, and the fixed width makes this button seemed cramped. My opinion is that you should split the button/input with a <Divider/>, and add more spacing.